data/json/mutations/mutations.json

```diff
@@ -5211,7 +5214,7 @@
     "visibility": 4,
     "ugliness": 3,
     "mixed_effect": true,
-    "description": "Your paws are much larger now.  Manual dexterity is difficult: permanent hand encumbrance of 20, serious problems crafting, and no gloves.  But you can swim more effectively.",
+    "description": "Your paws are much larger now.  Manual dexterity is difficult: permanent hand encumbrance of 20, serious problems crafting, and no gloves.  But, you can swim more effectively.",
```

Comment:
```suggestion
    "description": "Your paws are much larger now.  Manual dexterity is difficult: permanent hand encumbrance of 20, serious problems crafting, and no gloves.  But you can swim more effectively.",
```
Why the comma?

Reply:
Natural reading flow. If you speak the sentence aloud, you can practically hear the comma there.

Reply:
Hrm. When I say it, it's more like "no gloves - but you can swim more effectively" (or perhaps simply "swim better"?

Reply:
"In return, though, you can swim better"?